universal-login-wallet/cypress/integration/createAccount.test.ts

```diff
@@ -16,4 +17,17 @@ describe('E2E: Create Account', () => {
       cy.checkLocalStorageTokenProp('wallet-ganache', 'kind', 'Deployed');
     });
   });
+
+  ['macbook-13', 'iphone-x', 'iphone-5', 'ipad-2'].forEach((size, index) => {
+    it(`Check devices modal on ${size}`, () => {
+      cy.viewport(size);
+      cy.goToDevices();
+      cy.ensureCorrectLocation('/dashboard/devices');
+      cy.checkDevicesList();
+      cy.goToDisconnectAccount();
+      cy.ensureCorrectLocation('/dashboard/devices/disconnectAccount');
+      cy.checkDeleteAccount();
```

Comment:
I would move it to the previous test. This will require renaming the title of this test from 'Create account ..` to `Wallet roundtrip`. Then we will have one test that will click through the whole application.